format/apple/bplist/ns_keyed_archiver.jq

```diff
@@ -0,0 +1,77 @@
+def from_ns_keyed_archiver(root):
+  (  
+    . as 
+      {
+        "$objects": $objects
+      }
```

Comment:
"fq-jq-style" would be this i think:
```jq
  ( . as {
      "$objects": $objects
    }
  | ...
```
But maybe we can skip fixing style stuff and i can re-format things later. Have thought about doing a jqfmt thingy but it's quite tricky :)